ios/KMMIC/Sources/Presentation/Navigator/NavigatorStack.swift

```diff
@@ -30,6 +30,9 @@ struct NavigatorStack: View {
                 case let .surveyQuestion(viewModel):
                     SurveyQuestionScreen(viewModel: viewModel)
                         .environmentObject(navigator)
+                case .thank:
+                    ThankScreen()
+                        .environmentObject(navigator)
```

Comment:
Does it necessary to pass `navigator` as the `enviromentObject` here? I'm asking because you're also passing it in line 38 below

Reply:
I faced the same issue with the line 28 🙏 
```
// FIXME: Navigator is not found if we don't add this
```